glossary.md

```diff
@@ -81,7 +81,7 @@ Nếu một từ chưa có trong bảng thuật ngữ dưới đây, các bạn
 | precision                         | precision                                                      |                                                              |
 | recall                            | recall                                                         |                                                              |
 | recognition                       | nhận dạng                                                      |                                                              |
-| regularization                    | regularization                                                 |                                                              |
+| regularization                    | ràng buộc                                                      |                                                              |
```

Comment:
Không đúng rồi em. Ràng buộc là "constrain". Trong tối ưu, hai khái niệm này là hoàn toàn khác nhau. regularization thường được dùng để bỏ đi các ràng buộc trong các bài toán tối ưu.

Reply:
@tiepvupsu em chưa nghĩ đến trường hợp đó. Em thấy #51 dịch từ này là "chính quy hóa". Anh thấy sao?

Reply:
Ở https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/pull/192#discussion_r341468863 có một cách dịch khá ổn là "thành phần kiểm soát"

Reply:
Anh thấy cũng ổn, @ngcthuong có ý kiến gì không?

Reply:
Đúng như Tiệp nói là regularization là chuyển constrain về dạng dễ giải được hơn và nó là 2 phương trình khác nhau và cách giải cũng khác nhau. 

Tuy nhiên cả hai đều có chung ý nghĩa là giới hạn không gian nghiệm thay vì ở dạng cố định chính xác như ở constrain thì linh động và dễ giải hơn với regularization. 

Về tài liệu tham khảo thì regularization được dịch là
- sự chính quy hoá
- sự quy tắc hóa

nhưng mà mình nghe thấy vẫn chưa chuẩn lắm. Như ý ở trên, do regularization thường đi kèm với thông số heuristic để điều chỉnh mức độ ràng buộc. Nên mình có thể gợi ý là 
- (phần) điều chuẩn 
- (phần Điều kiểm

Reply:
Đúng là *Chính quy hoá* dường như đang được dùng phổ biến. A vote *điều chuẩn*.

Reply:
Em cũng vote "điều chuẩn". Nhưng em muốn chỉnh lại:
* "Điều chuẩn" -> "regularize" 
* "Thành phần điều chuẩn" -> "Regularization" 

vd: "L2 regularization" -> "thành phần điều chuẩn L2"

Reply:
Mình tán thành với @rootonchair  về chia làm hai trường hợp cho động từ và danh từ. Mình góp ý thêm là phần danh từ không cần thêm từ "thành" vẫn đủ nghĩa cho nó ngắn gọn hơn một chút. 

- Regularize --> Điều chuẩn
- Regularization --> Phần điều chuẩn

Reply:
@ngcthuong mình thấy không khác nhau, nhưng khi ở trong một tổng, người ta thường dùng "thành phần thứ nhất", "thành phần thứ hai". Trong một hàm mất mát, mình có thể gọi số hạng cuối cùng là "thành phần điều chuẩn".

Reply:
Vì quan điểm của mình là từ khoá nên ngắn gọn mà vẫn giữ nghĩa. 

Nếu ngắn gọn không ổn - có khả năng nhầm lẫn theo nghĩa khác thì mình thêm từ.

Như ví dụ  cậu lấy,  mình thấy cũng OK khi thêm từ “phần” vào.

Reply:
"phần điều chuẩn" có thể dùng trong ngữ cảnh thường và "thành phần điều chuẩn" có thể dùng trong ngữ cảnh toán học nên em nghĩ mình nên chấp nhận cả hai. Em sẽ sửa lại và cập nhật thêm  "regularize". Mọi người kiểm tra nếu không ổn thì cứ nói em.